src/response.rs

```diff
@@ -263,6 +263,14 @@ impl Into<http::Response> for Response {
     }
 }
 
+impl From<serde_json::Value> for Response {
+    fn from(json_value: serde_json::Value) -> Self {
+        Response::new(StatusCode::Ok)
+            .body_json(&json_value)
+            .unwrap_or_else(|_| Response::new(StatusCode::InternalServerError))
```

Comment:
I'm not actually sure if this is reachable, since a serde_json::Value should basically be the safest thing to turn into json, since it's the intermediate representation before stringification

Reply:
we should probably use the `Status` trait here instead so we don't lose the original error message.

Reply:
As far as returning status: From<serde_json::Value for Response needs to return a Response, not a tide::Error. I can imagine the value of adding an Option<Error> on Response (or using ext in a systematic way) and adding a TryInto or downcast, but that's way beyond the scope of this PR. If people want to retain the original error message from transforming a json literal into a Response, they'll need to explicitly do the `body_json` call for now